tests/cross_functional/kcs/test_monitor_recovery.py

```
@@ -533,48 +599,94 @@ def deployments_to_revert(self):
+                    break
+                key = None
+                caps = []
+                if block.startswith("key ="):
+                    key = block.split(" = ")[1].strip()
+                elif "caps" in block:
+                    caps.append(block.strip())
+                if key:
+                    logger.info("Found key entry in keyring data")
+                    formatted_data.append(f"    key = {key}")
+                for cap in caps:
+                    logger.info(f"Found cap :{cap}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)